CMakeLists.txt

```diff
@@ -39,13 +39,36 @@ if(WIN32)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${C_FLAGS}")
 endif()
 
-set(CMAKE_EXPORT_DIR "lib/cmake/lizard")
```

Comment:
why did you remove CMAKE_EXPORT_DIR?

Reply:
Oops that was not intended, I put it back.